internal/anelace/argparse.go

```diff
@@ -179,11 +186,7 @@ func NewFromArgv(argv []string) (anl *Anelace) {
 		argParseErrs = append(argParseErrs, "The value of --hash-bits must be a minimum of 128 and be divisible by 8")
 	}
 
-	if !cfg.optSet.IsSet("inline-max-size") &&
-		!cfg.optSet.IsSet("ipfs-add-compatible-command") &&
-		cfg.requestedCollector != "none" {
-		argParseErrs = append(argParseErrs, "You must specify a valid value for --inline-max-size")
-	} else if cfg.InlineMaxSize < 0 ||
+	if cfg.InlineMaxSize < 0 ||
```

Comment:
You might want to rip out the entire `--ipfs-add-compatible-command`, or alternatively you need to fix it up a bit.

The way it is currently written it assumes that some of the defaults you did set above are *NOT* supplied. As it stands no matter what you supply to it will not touch the stuff you did on lines 120~130

So either remove it or move the defaults-application to the very end, after `ipfs-add-compatible-command` has been processed

Reply:
Removed the command